src/platform/site-wide/cta-widget/helpers.js

```diff
@@ -25,6 +26,7 @@ export const widgetTypes = {
   VETERAN_ID_CARD: 'vic',
   VIEW_APPOINTMENTS: 'view-appointments',
   VIEW_DEPENDENTS: 'view-dependents',
+  ADD_MODIFY_DEPENDENTS: 'add-remove-dependents',
```

Comment:
Did you mean to make `MODIFY` => `REMOVE` or vice versa for the value?

Reply:
Ah good catch. I'll update it.